src/components/internal/BaseTabs/index.ts

```diff
@@ -0,0 +1 @@
+export * as BaseTabsEnums from './BaseTabs.enums';
```

Comment:
I would rename the internal folder like `internal -> _internal` that would cause two things:
1. it will be clear on first sight that this is special folder that's for internal use only, because we already using couple folders starting with lowercase like `forms`, `layout` ...
2. it will move the folder to the top in folder structure so it will be easier to find it for us

Reply:
I Agree with that.